sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/src/Bindings/WebPubSubRequest.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net;
+using System.Net.Http;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.WebPubSub
+{
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class WebPubSubRequest
+    {
+        /// <summary>
+        /// Common request information from headers.
+        /// </summary>
+        public ConnectionContext ConnectionContext { get; }
+
+        /// <summary>
+        /// Request body.
+        /// </summary>
+        public ServiceRequest Request { get; internal set; }
+
+        /// <summary>
+        /// Flag to indicate whether it's an <see href="https://github.com/cloudevents/spec/blob/v1.0/http-webhook.md#4-abuse-protection">Abuse Protection</see> request.
+        /// </summary>
+        public bool IsAbuseRequest { get; internal set; }
```

Comment:
Consider calling it `IsValidationRequest` instead. It might more clearly describe what it is.

Reply:
How about naming as `IsPing` cause this is a request similar like ping regularly from service to server for abuse check. And short name makes it simple for javascript language users.

Reply:
updated to `IsValidationRequest` and use type `ValidationRequest` to help c# user to work with directly.